README.md

```diff
@@ -1,162 +1,138 @@
-- `EXP_ID`: Atlas Experiment identifier.
-- `EXPERIMENT_MATRICES_PATH`: The path to the directory where the `$EXP_ID/.expression_tpm.mtx[|_cols|_rows].gz` matrix market files reside.
-- `dbConnection`: A postgres db connection string of the form `postgresql://{user}:{password}@{host:port}/{databaseName}` pointing to a Postgres 10 server where the expected `scxa_analytics` table exists.
-
-Additionally, it is recommended that `bin` directory on the root is prepended to the `PATH`. Then execute:
+It is recommended that `bin` is prepended to the `PATH`.
 
+### Delete data
+Set `dbConnection` and `EXP_ID`, then run `delete_db_scxa_analytics.sh`:
 ```bash
-load_db_scxa_analytics.sh
-```
-
-## Delete data for experiment
-
-Set the desired database connection in `dbConnection` and experiment accession in `EXP_ID` and use `delete_db_scxa_analytics.sh`:
-
-```bash
-export EXP_ID=TEST-EXP1
+export EXP_ID=...
 export dbConnection=...
 
 delete_db_scxa_analytics.sh
 ```
 
-# `scxa_coords` Table
-
-## Create schema
-
-Same as in `scxa_analytics` currently.
-
-## Load data
+## `scxa_coords` and `scxa_dimension_reduction` tables
 
-The main executable is `bin/load_db_scxa_dimred.sh`, which requires the following environment variables to be set:
-- `EXP_ID`: Atlas experiment identifier.
-- `EXPERIMENT_DIMRED_PATH`: path to directory containing tsne and umap files for loading. Files are expected to have the structure <prefix><perplexity_number><suffix> (tsne) or <prefix><neighbors_number><suffix> (umap) with the default suffix and prefix defined in the script. These can be configured from outside through env vars.
-- `dbConnection`: A postgres db connection string of the form `postgresql://{user}:{password}@{host:port}/{databaseName}` pointing to a Postgres 10 server where the expected `scxa_coords` table exist.
+### Load data
+Run `bin/load_db_scxa_dimred.sh`. It requires the following environment variables:
 
-Additionally, it is recommended that `bin` directory on the root is prepended to the `PATH`. Then execute:
+| Variable name       | Description                                                                                                                                                                                                        |
+|---------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `EXP_ID`            | Experiment accession                                                                                                                                                                                               |
+| `DIMRED_TYPE`       | The dimension reduction type, such as "umap" or "tsne"; the value is arbitrary and supplied by the user                                                                                                            |
```

Comment:
`such as "umap" or "tsne"; the value is arbitrary and supplied by the user`, not sure we need to mention in the capitals 'UMPA' and 't-SNE' as per the database.

Reply:
Since the value is arbitrary there’s no need to restrict it to lowercase. I’ll include your suggestion.